src/global/project/mod.rs

```diff
@@ -1070,6 +1077,88 @@ impl Project {
         );
         Ok(state_changes)
     }
+
+    /// Install the menu items of the environment
+    pub async fn install_menu_items(
+        &self,
+        env_name: &EnvironmentName,
+    ) -> miette::Result<StateChanges> {
+        let mut state_changes = StateChanges::default();
+        let environment = self
+            .environment(env_name)
+            .ok_or_else(|| miette::miette!("Environment {} not found", env_name.fancy_display()))?;
+
+        if environment.menu_install() {
+            // Find menu items in the prefix
+            let prefix = self.environment_prefix(env_name).await?;
+            let menu_files = prefix.find_menu_schema_files().await?;
+
+            // Install menu items
+            for menu_file in menu_files {
+                // TODO: Make the mode configurable
+                match rattler_menuinst::install_menuitems(
+                    menu_file.as_path(),
+                    prefix.root(),
+                    prefix.root(),
+                    environment.platform().unwrap_or(Platform::current()),
+                    MenuMode::User,
+                ) {
+                    Ok(_) => {
+                        tracing::info!("Installed menu item: {}", menu_file.display());
+                        state_changes.insert_change(
+                            env_name,
+                            StateChange::InstalledMenuItem(
+                                menu_file
+                                    .file_name()
+                                    .unwrap_or("menu_file".as_ref())
+                                    .to_string_lossy()
+                                    .to_string(),
+                            ),
+                        );
+                    }
+                    // Don't fail on menu install errors, menuinst is too unstable to break the whole process because of issue with it.
```

Comment:
What makes menuinst so unstable that we wouldn't error on it?

Reply:
This might be something we want to test, but as it was a prototype state feature I didn't want to block users with it.